src/OrleansTestingHost/TestingSiloOptions.cs

```diff
@@ -46,6 +50,8 @@ public class TestingSiloOptions
         public bool ParallelStart { get; set; }
         public GlobalConfiguration.ReminderServiceProviderType ReminderServiceType { get; set; }
         public string DataConnectionString { get; set; }
+        public Action<ClusterConfiguration> ConfigurationCustomizer { get; set; }
```

Comment:
I would recommend separating all those changes to Test framework into a separate PR. I think some of them are duplicating some other functionality we already recently added (AdjustConfiguration). Even if it is not duplicating, it would be easier to get convinced we indeed need all these Test framework changes if they are separated,